styles.css

```
@@ -0,0 +1,75 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #f4f4f4;
+}
+
+.parent {
+    display: grid;
+    grid-template: auto 1fr auto / auto 1fr auto;
+}
+
+header {
+    display: flex;
+    padding: 20px;
+    border-bottom: 2px solid #333; 
+    padding-bottom: 20px;
+    align-items: center;
+    justify-content: flex-start; 
+    padding-left: 20px;
+    padding-right: 20px;
+}
+
+header img {
+    margin-right: 15px; 
+}
```

```suggestion
header {
    display: flex;
    padding: 20px;
    border-bottom: 2px solid #333; 
    padding-bottom: 20px;
    align-items: center;
    justify-content: flex-start; 
    padding-left: 20px;
    padding-right: 20px;
    
    img {
        margin-right: 15px; 
    }
}
```

Recuerde que puede usar CSS Nesting